### DISCUSSION TITLE
Snow Depth Display Mode?

### DISCUSSION THREAD

**Discussion by wilburcoding** [niels747/2D-Weather-Sandbox#28] [Ideas]
Possible feature suggestion: A snow depth display mode showing how much snow is on the ground - this could be pretty useful in showing what areas received the most snow.

**Comment by Meso22** [comment]
Weather Stations are being added so I think there will also be snow depth along with it.

**Comment by niels747** [comment]
Good idea, but I don't know what the best way to show snow depth is...

**Comment by niels747** [comment]
Any sugestions on how exactly you would like it to look?

**Reply by kidnamednuclearwarin72hours** [reply]
maybe a snow depth view mode that uses a colour gradient?

**Reply by wilburcoding** [reply]
maybe a layer of color above land representing snow depth?
